sdk/identity/azure-identity/test/ut/workload_identity_credential_test.cpp

```diff
@@ -26,10 +26,12 @@ struct TempCertFile final
 TEST(WorkloadIdentityCredential, GetCredentialName)
 {
   TempCertFile const tempCertFile;
```

Comment:
Test that the options are getting populated from the environment - use `CredentialTestHelper::EnvironmentOverride`, it is used in `environment_credential_test` and `managed_identity_credential_test`